frontend/packages/kubevirt-plugin/integration-tests/tests/mocks/vmBuilderPresets.ts

```diff
@@ -87,18 +87,18 @@ export const computedVMBuilders = deepFreeze(
   ),
 );
 
-export const getBasicVMBuilder = () =>
+export const getBasicVMBuilder = (namespace?: string) =>
   new VMBuilder()
-    .setNamespace(testName)
+    .setNamespace(namespace || testName)
```

Comment:
could we call setNamespace 2nd time on the returned builder instead?

It could get quite confusing if start introducting additional params here

Reply:
yes, it's better to call `setNamespace` instead of changing here.